### ISSUE TITLE
Test fails when Accessibility Inspector is off

### ISSUE THREAD

**Issue by datinc** [kif-framework/KIF#889]
When running our Unit Test on CI environment, test would fail with

`  Assertions: The step timed out after 10.00 seconds: Failed to find accessibility element with the label "Some Label"
  File: SomeClass.m:123`

When I enabled the accessibility inspector using `Settings > General > Accessibility > Accessibility Inspector`, the test all passed.

**Comment by phatmann** [comment]
KIF has code to turn on the accessibility inspector. What version of iOS are you using?

**Comment by datinc** [comment]
Xcode Version 7.3.1
iOS 9.1

**Comment by phatmann** [comment]
Does it work when you run in locally?

**Comment by datinc** [comment]
Locally the same thing is happing
* Passes with accessibility inspector turned on.
* Fails with accessibility inspector turned off.

**Comment by phatmann** [comment]
What KIF version are you using?

**Comment by datinc** [comment]
In my Podfile I have imported it using:
`pod 'KIF/IdentifierTests', '~> 3.0'`

In Podfile.lock you can see that it is 3.4.2
```
  - KIF/Core (3.4.2)
  - KIF/IdentifierTests (3.4.2):
    - KIF/Core
```

**Comment by phatmann** [comment]
There have been a lot of fixes. Can you try the HEAD version? If that works I will push a new release ASAP.

**Comment by datinc** [comment]
On it

**Comment by datinc** [comment]
`pod 'KIF/IdentifierTests', :git => 'https://github.com/kif-framework/KIF.git'`
Still fails.

**Comment by datinc** [comment]
I can reproduce this in a sample project. The steps to reproduce this error:
1. Make sure Accessibility Inspector is turned off on the simulator
2. Re-start the simulator
3. Run test

**Comment by phatmann** [comment]
KIF is supposed to turn on the Accessibility Inspector for you. Can you please debug KIFAcessibilityEnabler to see what is failing?

**Comment by datinc** [comment]
`KIFEnableAccessibility()` is never called in my code. I can see that is because my test case class does not inherit from KIFTestCase. Once I make all my test case classes inherit from this, everything works.

**Comment by phatmann** [comment]
Glad you found the issue.

**Comment by datinc** [comment]
Would this be a bug or a documentation issue. 

> KIFTestCase is not necessary for running KIF tests. Tests can run directly in XCTestCase or any subclass. The basic requirement is that when you call tester or system, self must be an instance of XCTestCase.

It seems to be necessary.

**Comment by phatmann** [comment]
Good point. I just created ticket #890 for this issue.

**Comment by datinc** [comment]
Thanks for all the help @phatmann!
